code/frameworks/nextjs/package.json

```diff
@@ -59,13 +59,15 @@
     "prep": "../../../scripts/prepare/bundle.ts"
   },
   "dependencies": {
+    "@babel/preset-typescript": "^7.18.6",
```

Comment:
Was this addition intended?

Reply:
doubtful, likely a merge conflict resolved incorrectly